tests/explainers/test_deep.py

```diff
@@ -253,7 +253,7 @@ def test_tf_keras_imdb_lstm(random_seed):
     diff = sess.run(mod.layers[-1].output, feed_dict={mod.layers[0].input: testx})[0, :] - sess.run(
         mod.layers[-1].output, feed_dict={mod.layers[0].input: background}
     ).mean(0)
-    np.testing.testing_allclose(sums, diff, atol=1e-02), "Sum of SHAP values does not match difference!"
+    np.testing.assert_allclose(sums, diff, atol=1e-02), "Sum of SHAP values does not match difference!"
```

Comment:
I think this was a typp that crept through in #3555 . The test is skipped for recent tensorflow versions so it slipped through the net.